### PR TITLE
Webhooks refresh

### PR DESCRIPTION
**Pull request by taylorpwhipp**
This PR includes changes for the webhooks guide.  It's meant to make things a bit more "straight to the point" and provide some extra context around configuring the webhook.

@pierrelevaillant can you please let me know if the arrows that I used are bad design/ you'd prefer something else to indicate where to click?   Happy to do a walkthrough on how you would do this in Figma, so I don't have to ask next time :) 

https://github.com/liveblocks/liveblocks.io/issues/904

[Preview branch](https://liveblocks-io-git-webhooks-update-liveblocks.vercel.app/docs/guides/webhooks)

### CODE REVIEW

**Comment by pierrelevaillant** [comment]
@taylorpwhipp Did you do this in Figma somewhere? I'm going to explore arrows look and feel!

**Review by pierrelevaillant — CHANGES_REQUESTED** [review]
Videos are great but they render weird to me. It seems the colors are damaged (we can especially see the difference by comparing the background color). Can we fix that? Happy to help debugging the record settings.

Rest of the guide is great, the formatting is really clean 👍

File: docs/pages/guides/webhooks.mdx [hunk 1]
```diff
@@ -2,98 +2,296 @@
...
-## What is a webhook?
+If you would like to create an endpoint to receive webhook events, you will do
+so from within the webhooks dashboard for your project.
 
-Webhooks pass information from one application to another instantaneously. A
-webhook is simply an HTTP callback triggered by an event. When an application
-subscribes to an event, and that event occurs, the webhook invokes the HTTP
-callback with a URL you specify (called the webhook endpoint). Webhooks
-streamline the collection of data between two connected services in real-time.
+1. From the [dashboard](/dashboard) overview, navigate to the project you would
```
Comment (adigau) [inline docs/pages/guides/webhooks.mdx:28]: Not new to this PR, but we should find a way to have those links open in a new tab not to break navigation.
Maybe a solution exists? See [this thread](https://stackoverflow.com/questions/4425198/can-i-create-links-with-target-blank-in-markdown)


cc @pierrelevaillant

File: docs/pages/guides/webhooks.mdx [hunk 2]
```diff
@@ -2,98 +2,296 @@
...
+is enabled by default. In addition, you should also follow the steps in the
+security verification section before sending events.
+
+</Banner>
+
+### Subscribing to events [#subscribing-to-events]
+
+Each endpoint you provide in the webhooks dashboard listens to all events by
+default but can be easily configured to only listen to a subset by updating the
+Message Filtering section after endpoint creation.
```
Comment (adigau) [inline docs/pages/guides/webhooks.mdx:78]:
```suggestion
By default, each endpoint provided in the webhooks dashboard listens to all events. However, you can easily configure it to only listen to a subset of events by updating the "Message Filtering" section after creating the endpoint.
```

The phrase was quite long, could we make it shorter?

Also, not sure if that’s better, but shouldn't we have the section name in quotes? Like we do when referring to buttons?

File: docs/pages/guides/webhooks.mdx [hunk 3]
```diff
@@ -2,98 +2,296 @@
...
+const signature = crypto
+  .createHmac("sha256", secretBytes)
+  .update(signedContent)
+  .digest("base64");
+```
+
+#### 3. Validate the signature [@hidden]
+
+This generated signature should match one of the ones sent in the
+`webhook-signature` header.
```
Comment (adigau) [inline docs/pages/guides/webhooks.mdx:243]:
```suggestion
The generated signature should match one of the signatures sent in the `webhook-signature` header.
```

one of the ones felt weird

File: docs/pages/guides/webhooks.mdx [hunk 4]
```diff
@@ -2,98 +2,296 @@
...
+
+The webhook-signature header comprises a list of space-delimited signatures and
```
Comment (adigau) [inline docs/pages/guides/webhooks.mdx:245]:
```suggestion
The `webhook-signature` header comprises a list of space-delimited signatures and
```

to be coherent with the rest, no?

File: docs/pages/guides/webhooks.mdx [hunk 5]
```diff
@@ -2,98 +2,296 @@
...
 
 - `MaxConnectionsReached`
 
 #### UserEnteredEvent
 
-A user enters a room when they connect to a room, which is when this event is
-triggered. The `numActiveUsers` field represents the number of users in the room
-after the user has entered. This event is not throttled.
+When a user connects to a room, an event is triggered, indicating that the user
+has entered. The numActiveUsers field shows the number of users in the room
```
Comment (adigau) [inline docs/pages/guides/webhooks.mdx:293]:
```suggestion
has entered. The `numActiveUsers` field shows the number of users in the room
```

File: docs/pages/guides/webhooks.mdx (hunk 1 above)
Comment (pierrelevaillant) [reply docs/pages/guides/webhooks.mdx:28]: `<a href="/dashboard" target="_blank">…</a>`

File: docs/pages/guides/webhooks.mdx [hunk 6]
```diff
@@ -166,13 +364,12 @@ const userLeftEvent = {
 #### StorageUpdatedEvent
 
 Storage is updated when a user writes to storage. This event is throttled at
-five minutes and, as such, may not be triggered for every write. The `updatedAt`
-field is the time of the last write.
+five minutes and, as such, may not be triggered for every write.
 
-For example, if a user writes to storage at 1:00pm, the `storageUpdatedEvent`
+For example, if a user writes to storage at 1:00 pm, the `storageUpdatedEvent`
```
Comment (FlowFlorent) [inline docs/pages/guides/webhooks.mdx:369]:
```suggestion
For example, if a user writes to storage at 1:00 pm, the `StorageUpdatedEvent`
```

File: docs/pages/guides/webhooks.mdx [hunk 7]
```diff
@@ -166,13 +364,12 @@ const userLeftEvent = {
...
 event will be triggered shortly after. If the user writes to storage again at
-1:01pm, the `storageUpdatedEvent` event will be triggered 5 minutes after the
-first event was sent, around 1:05pm.
+1:01 pm, the `storageUpdatedEvent` event will be triggered 5 minutes after the
```
Comment (FlowFlorent) [inline docs/pages/guides/webhooks.mdx:371]:
```suggestion
1:01 pm, the `StorageUpdatedEvent` event will be triggered 5 minutes after the
```

File: docs/pages/guides/webhooks.mdx (hunk 2 above)
Comment (adigau) [reply docs/pages/guides/webhooks.mdx]: The changes make the phrase even longer, no? And what about making the "Message Filtering" section in quotes to be coherent when we talk about CTAs?

**Review by adigau — APPROVED** [review]

**Pull request merged**